contribution.md

```diff
@@ -0,0 +1,81 @@
+# Welcome to TCET Open Source contributing guide <!-- omit in toc -->
```

Comment:
`<!-- omit in toc -->` doesnt really work, since its being shown in the table of contents anyway
![image](https://github.com/tcet-opensource/erp-backend/assets/85873779/b564d0a7-0726-4191-96be-ec1c2709cc9d)
please remove that, or try to fix it